go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/longhorn/longhorn-instance-manager
 
-go 1.20
+go 1.21
+
+replace github.com/longhorn/go-spdk-helper => github.com/c3y1huang/go-spdk-helper v0.0.0-20231017050105-6c08295fcd89
```

Comment:
TODO: update after https://github.com/longhorn/go-spdk-helper/pull/48 is merged.